### PR TITLE
Home page tabs

### PR DESCRIPTION
**Pull request by patofernandez**
## Summary
A ViewPager with its respective TabLayout was added, a selector was created for each icon. Images were added
## Screenshot
image
## Trello
https://trello.com/c/xkh9LFYS

### CODE REVIEW

File: app/src/main/java/ar/com/wolox/android/example/ui/home/fragment/HomeFragment.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import kotlinx.android.synthetic.main.fragment_home.*
+import kotlinx.android.synthetic.main.fragment_viewpager.vViewPager
+import javax.inject.Inject
+
+class HomeFragment @Inject constructor() : WolmoFragment<BasePresenter<Any>>() {
+
+    @Inject internal lateinit var page1Fragment: NewsFragment
```
Comment (DemianIac) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/fragment/HomeFragment.kt]: You can create a companion object in NewsFragment with a method newInstance(). So you only need to call NewsFragment.newInstance() when you need it.

File: app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsPresenter.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import ar.com.wolox.wolmo.core.util.ToastFactory
+import javax.inject.Inject
+
+class NewsPresenter @Inject constructor(
+    private val toastFactory: ToastFactory
+) : BasePresenter<INewsView>() {
+
+    override fun onViewAttached() {
+        toastFactory.show("We don't show toasts from presenters")
```
Comment (DemianIac) [inline app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsPresenter.kt]: Nice message 😆

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -1,6 +1,46 @@
...
+            android:layout_marginLeft="8dp"
+            app:srcCompat="@drawable/wolox_logo_large" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_marginLeft="8dp"
+            android:textSize="30sp"
+            android:textColor="#000000"
```
Comment (DemianIac) [inline app/src/main/res/layout/fragment_home.xml]: Take the color from resources.

Comment (DemianIac) [reply app/src/main/res/layout/fragment_home.xml]: At this point maybe you should create a style for the text

File: app/src/main/res/layout/fragment_viewpager.xml [hunk 4]
```diff
@@ -1,11 +1,34 @@
...
     android:orientation="vertical">
 
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <ImageView
+            android:id="@+id/imageView2"
+            android:layout_width="70dp"
```
Comment (DemianIac) [inline app/src/main/res/layout/fragment_viewpager.xml]: This size should be in the resources file

**Review by DemianIac — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_home.xml (hunk 3 above)
Comment (patofernandez) [reply app/src/main/res/layout/fragment_home.xml]: done!

File: app/src/main/res/layout/fragment_viewpager.xml (hunk 4 above)
Comment (patofernandez) [reply app/src/main/res/layout/fragment_viewpager.xml]: done!

File: app/src/main/java/ar/com/wolox/android/example/ui/home/fragment/HomeFragment.kt (hunk 1 above)
Comment (patofernandez) [reply app/src/main/java/ar/com/wolox/android/example/ui/home/fragment/HomeFragment.kt]: done!

File: app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsPresenter.kt (hunk 2 above)
Comment (patofernandez) [reply app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsPresenter.kt]: lol done!

**Pull request merged**